source/inifiled.d

```diff
@@ -537,3 +536,39 @@ unittest {
 		assert(false);
 	}	
 }
+
+version(unittest) {
+    enum Check : string { disabled = "disabled"}
+
+    @INI
+    struct StaticAnalysisConfig
```

Comment:
inifiled style differs from phobos. opening curly braces go on the previous line